packages/server/src/api/controllers/user.js

```diff
@@ -63,7 +63,23 @@ exports.create = async function(ctx) {
   }
 }
 
-exports.update = async function() {}
+exports.update = async function(ctx) {
+  const db = new CouchDB(ctx.user.instanceId)
+  const user = ctx.request.body
+
+  const response = await db.put(user)
```

Comment:
dont' think we want to update the whole user here ? Unless we are always passing the username and password ? if so, then we need to hash the password before saving (see `create`). 

If this is the case, then `update` is probably the same as `create` ? ... i.e. `PUT /api/users` and `POST /api/users` should both point to `create` ?

Reply:
I fixed this by removing the pw from the object sent to the backend, leaves room for doing updates in the future to the object without too much hassle (I think=)

Reply:
Yea, i see what you've done... but i this will actually destroy wipe out the `password` field, as it is undefined ... and `db.put` will just replace the whole document ... i.e. whatever you pass to `db.put` will be the new record.

so... what you need to do, is something like

```
const user = db.get(ctx.request.body._id)
user.accessLevelId = ctx.request.body.accessLevelId
const response = db.put(user)
...
```

so, read the record first, update the fields that need updated, and save the record